tools/releases/helm.sh

```diff
@@ -16,13 +16,22 @@ GH_PAGES_BRANCH="gh-pages"
 function package {
   # First package all the charts
   for dir in "${CHARTS_DIR}"/*; do
-    if [ ! -d "$dir" ]; then
+    if [ ! -d "${dir}" ]; then
       continue
     fi
 
+    git diff HEAD --exit-code -- "${dir}"
+
+    # TODO remove this when Gateway is no longer experimental
+    if [[ "${BUILD_WITH_EXPERIMENTAL_GATEWAY}" != "true" && $(basename "${dir}") == "kuma" ]]; then
```

Comment:
In the build, we use `BUILD_WITH_EXPERIMENTAL_GATEWAY != N` to enable it, so we ought to be consistent. Maybe here and in the build, we enable gateway only if `BUILD_WITH_EXPERIMENTAL_GATEWAY=Y`?

Reply:
Changed it to be consistent. I like `!= N` because IMO it's not clear that it has to be `Y` to be true and if someone's setting it at all they likely want it to be true.

Reply:
Merging this as is